admin/app/components/organizations/information-section-edit.js

```diff
@@ -94,19 +94,49 @@ class Form extends Object.extend(Validations) {
   @tracked isManagingStudents;
   @tracked documentationUrl;
   @tracked showSkills;
+  @tracked identityProviderForCampaigns;
 }
 
 export default class OrganizationInformationSectionEditionMode extends Component {
   @service accessControl;
+  @service oidcIdentityProviders;
+
   @tracked isEditMode = false;
   @tracked showArchivingConfirmationModal = false;
+  noIdentityProviderOption = { label: 'Aucun', value: 'None' };
+  garIdentityProviderOption = { label: 'Médiacentre', value: 'GAR' };
 
   constructor() {
     super(...arguments);
     this.form = Form.create(getOwner(this).ownerInjection());
     this._initForm();
   }
 
+  get identityProviderOptions() {
+    const oidcIdentityProvidersOptions = this.oidcIdentityProviders.list.map((identityProvider) => ({
+      value: identityProvider.code,
+      label: identityProvider.organizationName,
+    }));
+    return [this.noIdentityProviderOption, this.garIdentityProviderOption, ...oidcIdentityProvidersOptions];
+  }
```

Comment:
pourquoi ne pas jumeler les 2 ensembles genre

```suggestion
  noneAndOtherIdentityProviderOptions = [{ label: 'Aucun', value: 'None' }, { label: 'Médiacentre', value: 'GAR' }];

  constructor() {
    super(...arguments);
    this.form = Form.create(getOwner(this).ownerInjection());
    this._initForm();
  }

  get identityProviderOptions() {
    const oidcIdentityProvidersOptions = this.oidcIdentityProviders.list.map((identityProvider) => ({
      value: identityProvider.code,
      label: identityProvider.organizationName,
    }));
    return [...this.noneAndOtherIdentityProviderOptions, ...oidcIdentityProvidersOptions];
  }
```

Reply:
Je trouve ça plus clair de les avoir séparé , les noms sont plus explicites. 
Avec `otherIdentityProvider` on dirait qu'on en a plus que 1, alors qu'on a que le GAR actuellement. Je sais qu'on peut imaginer qu'on en rajoutes d'autres par la suite mais : ce n'est pas prévu (on privélégiera que les interco OIDC), et si ça arrive, autant modifier / factoriser à ce moment là.
En plus le `noIdentityProviderOption` est utilisé plus bas dans le `onChangeIdentityProvider` car dans le cas de cette options on veut renvoyer `null` à l'API